docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -0,0 +1,142 @@
+
+- In the **Name** field, enter the name for the YugabyteDB universe (or example, yb-k8s).
+
+- Use the **Provider** field to select the appropriate Kubernetes cloud (or example, K8s Provider). Notice that additional fields are now displayed, as shown in the following illustration:
+
+   ![img](/images/yb-platform/kubernetes-config4.png)
+
+Complete the rest of the **Cloud Configuration** section as follows:
+
+- Use the **Region** field to select the region. This enables the **Availability Zones** option that allows you to see zones belonging to that region. 
+
+- Provide the value in the **Pods** field. This value should be equal to or greater than the replication factor. The default value is 3. When this value is supplied, the pods are automatically placed across all the availability zones to guarantee the maximum availability.
+
+- In the **Replication Factor** field, define the replication factor. The default is 3. 
+
+  ![img](/images/yb-platform/kubernetes-config5.png)
+
+Complete the **Instance Configuration** section as follows:
+
+- Use the **Instance Type** field to select the CPU and memory combination, as per needs to allocate the YB-TServer pods. The default is Small (4 Cores & 7.5GB RAM). You can override this setting when you configure the Kubernetes cloud provider (see [Configuring the region and zones](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-region-and-zones)).
+
+- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1*100GB.
+
+- Use the **Enable YSQL** field to enable the YSQL API endpoint to run Postgres-compatible workloads. This setting is enabled by default.
+
+- Use the **Enable YEDIS** field to enable the YEDIS API endpoint to run REDIS-compatible workloads. This setting is disabled by default.
+
+- Use the **Enable Node-to-Node TLS** field to enable encryption-in-transit for communication between the database servers. This setting is disabled by default.
+
+- Use the **Enable Client-to-Node TLS** field to enable encryption-in-transit for communication between clients and the database servers. This setting is disabled by default.
+
+- Use the **Enable Encryption at Rest** field to enable encryption for data stored on the tablet servers. This setting is disabled by default.
+
+  ![img](/images/yb-platform/kubernetes-config6.png)
+
+Complete the **Advanced** section as follows:
+
+- In the **DB Version** field, specify the YugabyteDB version. The latest version is the default.
+
+- Use the **Enable IPV6** to enable the use of IPV6 networking for connections between the database servers. This setting is enbled by default.
+
+- Use the **Enable Public Network Access** field to assign a load balancer or nodePort for connecting to the database endpoints over the internet. This setting is enbled by default.
+
+  ![img](/images/yb-platform/kubernetes-config7.png)
```

Comment:
Just a new image as per the latest Platform version.

![image](https://user-images.githubusercontent.com/13635843/123227845-9ad8b680-d4f2-11eb-9d18-c67222a429fa.png)

Reply:
Replaced image